controller/Review.php

```diff
@@ -298,4 +312,51 @@ private function isSubmissionReviewRequestMessageProvided(): bool
 
         return $messageType === LtiMessageInterface::LTI_MESSAGE_TYPE_SUBMISSION_REVIEW_REQUEST;
     }
+
+    private function getReviewPanelConfig(): array
+    {
+        $extensionsManager = $this->getServiceLocator()->get(common_ext_ExtensionsManager::SERVICE_ID);
+        $extension = $extensionsManager->getExtensionById('ltiTestReview');
+        return $extension->getConfig('ReviewPanel');
+    }
+
+    private function getDisplaySectionTitlesOption(LtiLaunchData $launchData): bool
+    {
+        $reviewPanelConfig = $this->getReviewPanelConfig();
+        $extensionSectionTitles = $reviewPanelConfig[self::OPTION_DISPLAY_SECTION_TITLES];
+
+        $ltiParamSectionTitles = $launchData->hasVariable(self::LTI_DISPLAY_SECTION_TITLES)
+            ? $launchData->getVariable(self::LTI_DISPLAY_SECTION_TITLES)
+            : null;
+
+        // $sectionTitles priority: LTI param > extension config > true
+        $sectionTitles = true;
+        if (isset($extensionSectionTitles)) {
+            $sectionTitles = $extensionSectionTitles;
+        }
+        if (isset($ltiParamSectionTitles)) {
+            $sectionTitles = $ltiParamSectionTitles;
+        }
+        return filter_var($sectionTitles, FILTER_VALIDATE_BOOLEAN);
```

Comment:
Just simplifying a bit:

```
        $fromLti = $launchData->hasVariable(self::LTI_DISPLAY_SECTION_TITLES)
            ? $launchData->getVariable(self::LTI_DISPLAY_SECTION_TITLES)
            : null;

        if ($fromLti !== null) {
            return filter_var($fromLti, FILTER_VALIDATE_BOOLEAN);
        }

        return filter_var($this->getReviewPanelConfig()[self::OPTION_DISPLAY_SECTION_TITLES] ?? null, FILTER_VALIDATE_BOOLEAN);
```

Reply:
I get the idea (and appreciate your help), but I think I wrote it quite easily readable already.
I'm short on time, so I'm reluctant to refactor this and test all over again - I'll take the comment on board for next time.